clang/test/CodeGenSYCL/dead-elim-sycl.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang -cc1 -triple spir64-unknown-unknown  -fsycl-is-device  -emit-llvm-bc  -O1 -fno-legacy-pass-manager -fdebug-pass-manager -fenable-sycl-dae -o /dev/null -x c++ < %s 2>&1 | FileCheck %s
```

Comment:
How about adding one more run line that tests with -fdisable-llvm-passes?

Reply:
Done